cli-setup.sh

```diff
@@ -257,11 +265,11 @@ create_cli_config(){
 	    stdout_log "Attempt $auth_retry of ${max_auth_retry}:"; fi
 
     if [ ! -z "${PF9_MGMTURL}" ] && [ ! -z "${PF9_USER}" ] && [ ! -z "${PF9_PASS}" ] && [ ! -z "${PF9_REGION}" ] && [ ! -z "${PF9_TENANT}" ]; then
-        eval "${cli_exec}" config create --du_url ${PF9_MGMTURL} --os_username ${PF9_USER} --os_password ${PF9_PASS} --os_region ${PF9_REGION} --os_tenant ${PF9_TENANT}
+        eval "${cli_exec}" config create --du_url ${PF9_MGMTURL} --os_username ${PF9_USER} --os_password ${PF9_PASS} --os_region ${PF9_REGION} --os_tenant ${PF9_TENANT} ${dev_key} ${disable_analytics}
```

Comment:
May be OK for now. But we should explore the option of creating a different config file for these things. Let the user created config only deal with user <-> DU related information.

Reply:
Yea I agree, but if we are considering to rewrite pmk cli, maybe its ok to ignore this for now.